app/wyzebridge/mtx_server.py

```diff
@@ -137,7 +142,9 @@ def start(self):
         if self.sub_process:
             return
         logger.info(f"[MTX] starting MediaMTX {getenv('MTX_TAG')}")
-        self.sub_process = Popen(["/app/mediamtx", "/app/mediamtx.yml"])
+        self.sub_process = Popen(
+            ["./mediamtx", "./mediamtx.yml"], stdout=None, stderr=None
```

Comment:
Is there a reason to add `stdout=None` and `stderr=None`, considering the fact it wasn't there originally?

Reply:
Yes, without that you don't see the output from MediaMTX in the docker output